packages/react/vite.config.ts

```diff
@@ -0,0 +1,72 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+import dts from "vite-plugin-dts";
+import { viteStaticCopy } from "vite-plugin-static-copy";
+import { globSync } from "glob";
+import { PreRenderedChunk } from "rollup";
+
+const OUT_DIR = "lib";
+
+function names({ facadeModuleId }: PreRenderedChunk) {
+  if (facadeModuleId?.endsWith("src/index.ts")) {
+    return "index.js";
+  }
+
+  if (facadeModuleId?.includes("components")) {
+    const componentName = facadeModuleId.split("/").at(-2);
+
+    return `_store/${componentName}-entry-[hash].js`;
```

Comment:
Why is there _store in the path name?

Reply:
Just for the build structure, as mentioned in the PR description